doc/tutorial/debug_faq.txt

```diff
@@ -214,10 +214,12 @@ The ``compute_test_value`` mechanism works as follows:
 * ``ignore``: Silently ignore the computation of intermediate test values, if a
   variable is missing a test value.
 
-.. note::
-  This feature is currently incompatible with ``Scan`` and also with ops
-  which do not implement a ``perform`` method.
+Theano also provides the ``print_test_value`` print flag which can hold a boolean value.The default value is ``false``.If ``True``, Theano will override the ‘__str__’ method of its variables to also print the tag.test_value when this is available.
+It can be used as 
```

Comment:
Please also try to be as concise as possible. What is important here is just to point out that setting the flag allows to recover the test value by accessing the variable, instead of its `tag.test_value` attribute. More details can be found in the description of the flag in the [config section](http://deeplearning.net/software/theano/library/config.html#config.print_test_value). 

While you are at it, please add the following example **to the config section**:
```python
import numpy
import theano
theano.config.compute_test_value='warn'
a = theano.tensor.matrix()
b = theano.tensor.scalar()
a.tag.test_value = numpy.array(range(2*3), dtype='float32').reshape((2,3))
b.tag.test_value = 2

# NORMAL BEHAVIOUR (print_test_value = False)
c = a * b
c 
    Elemwise{mul,no_inplace}.0 
c.tag.test_value
    array([[  0.,   2.,   4.],
           [  6.,   8.,  10.]], dtype=float32)
c.shape
    Shape.0
c.shape.tag.test_value
    array([2, 3]) 
 
# ENABLING PRINT_TEST_VALUE
theano.config.print_test_value=True
c
    Out: Elemwise{mul,no_inplace}.0
    array([[  0.,   2.,   4.],
           [  6.,   8.,  10.]], dtype=float32)
c.shape
    Shape.0
    array([2, 3]) 
```